backend/tracim_backend/tests/functional/test_preview_link.py

```diff
@@ -31,6 +31,21 @@
     "image": "http://example.invalid/article_thumbnail.jpg",
 }
 
+relative_image_url_opengraph_html = """
+<html>
+<header>
+<meta property="og:title" content="Example title of article">
+<meta property="og:site_name" content="example.invalid website">
+<meta property="og:type" content="article">
+<meta property="og:url" content="http://example.invalid/example-title-of-article">
+<meta property="og:image" content="article_thumbnail.jpg">
+<meta property="og:description" content="This example article is an example of OpenGraph protocol.">
+</header>
+<body>
+</body>
+</html>
+"""
+
```

Comment:
Is it possible to have test with  some opengraph property but no og:image ?

Reply:
I added one.